frame/assets-registry/src/lib.rs

```diff
@@ -173,6 +189,29 @@ pub mod pallet {
 			Self::deposit_event(Event::<T>::AssetUpdated { asset_id, location });
 			Ok(().into())
 		}
+
+		/// Minimal amount of asset_id required to send message to other network.
+		/// Target network may or may not accept payment.
+		/// Assumed this is maintained up to date by technical team.
+		/// Mostly UI hint and fail fast solution.
+		/// In theory can be updated by parachain sovereign account too.
+		/// If None, than it is well known cannot pay with that asset on target_parachain_id.
+		/// If Some(0), than price can be anything greater or equal to zero.
+		/// If Some(MAX), than actually it forbids transfers.
+		#[pallet::weight(<T as Config>::WeightInfo::set_min_fee())]
+		pub fn set_min_fee(
+			origin: OriginFor<T>,
+			target_parachain_id: ParaId,
+			asset_id: T::ForeignAssetId,
+			minimal_amount: Option<T::Balance>,
+		) -> DispatchResultWithPostInfo {
+			T::ParachainOrGovernanceOrigin::ensure_origin(origin)?;
+			// TODO: in case it is set to parachain, check that chain can target only its origin
+			MinFeeAmounts::<T>::mutate_exists(target_parachain_id, asset_id, |x| {
+				*x = minimal_amount
+			});
```

Comment:
I would add a event about success.This can be useful for UI and tests.

Reply:
@andor0 will add in next PR